content/pl/docs/concepts/overview/_index.md

```diff
@@ -1,7 +1,168 @@
 ---
-title: "Przegląd"
 weight: 20
-description: Ogólny zarys Kubernetesa i komponentów, z których jest zbudowany.
-sitemap:
-  priority: 0.9
+description: >
+ Kubernetes to przenośna, rozszerzalna platforma oprogramowania *open-source* służąca do zarządzania zadaniami i serwisami uruchamianymi w kontenerach. Umożliwia ich deklaratywną konfigurację i automatyzację. Kubernetes posiada duży i dynamicznie rozwijający się ekosystem. Szeroko dostępne są usługi, wsparcie i dodatkowe narzędzia.
+content_type: concept
+weight: 20
+card:
+  name: concepts
+  weight: 10
+no_list: true
 ---
+
+<!-- overview -->
+Na tej stronie znajdziesz ogólne informacje o Kubernetesie.
+
+
+<!-- body -->
+
+Kubernetes to przenośna, rozszerzalna platforma oprogramowania *open-source* służąca do zarządzania zadaniami i serwisami uruchamianymi w kontenerach,
+która umożliwia deklaratywną konfigurację i automatyzację. Ekosystem Kubernetesa jest duży i dynamicznie się rozwija.
+Usługi dla Kubernetesa, wsparcie i narzędzia są szeroko dostępne.
+
+Nazwa Kubernetes pochodzi z greki i oznacza sternika albo pilota.
+Skrót K8s powstał poprzez zastąpienie ośmiu liter pomiędzy "K" i "s".
+Google otworzyło projekt Kubernetes publicznie w 2014. Kubernetes korzysta z
+[piętnastoletniego doświadczenia Google w uruchamianiu wielkoskalowych serwisów](/blog/2015/04/borg-predecessor-to-kubernetes/)
+i łączy je z najlepszymi pomysłami i praktykami wypracowanymi przez społeczność.
+
+## Trochę historii
+
+Aby zrozumieć, dlaczego Kubernetes stał się taki przydatny, cofnijmy sie trochę w czasie.
+
+![Jak zmieniały sie metody wdrożeń](/images/docs/Container_Evolution.svg)
+
+**Era wdrożeń tradycyjnych:**
+Na początku aplikacje uruchamiane były na fizycznych serwerach. Nie było możliwości separowania zasobów poszczególnych aplikacji,
+co prowadziło do problemów z alokacją zasobów.
+Przykładowo, kiedy wiele aplikacji jest uruchomionych na jednym fizycznym serwerze,
+część tych aplikacji może zużyć większość dostępnych zasobów, powodując spowolnienie działania innych.
+Rozwiązaniem tego problemu mogło być uruchamianie każdej aplikacji na osobnej maszynie.
+Niestety, takie podejście ograniczało skalowanie, ponieważ większość zasobów nie była w pełni wykorzystywana,
+a utrzymanie wielu fizycznych maszyn było kosztowne.
+
+**Era wdrożeń w środowiskach wirtualnych:** Jako rozwiązanie zaproponowano wirtualizację, która umożliwiała
```

Comment:
Tutaj jest kilka zdań w czasie przeszłym, np `umożliwiała`, `wirtualizacja pozwalała`. Moze jednak zmienic to na `umożliwia/pozwala` albo `umożliwiła/pozwoliła`?